### ISSUE TITLE
Variable for used mail?

### ISSUE THREAD

**Issue by SirWild** [Healix/Gw2Launcher#274]
Heya 
there are some window caption variables like: %accountname%
Is there also a variable for the email used for the login?

**Comment by Healix** [comment]
No. Any option that supports variables will list all variables available to it. I'll add an %email% variable for the next release.

**Comment by SirWild** [comment]
Do you have a date for the next release?

**Comment by Healix** [comment]
I've been waiting for ArenaNet to release the CEF update. The next release would be within a few days of that.

If you want it now, I've added it to the [beta version](https://github.com/Healix/Gw2Launcher/raw/master/Gw2Launcher/bin64/Beta/Gw2Launcher.exe).

**Comment by SirWild** [comment]
Thanks! <3
